### PR TITLE
Adding place of work questions for NISRA HH/HI

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This adds two new questions with past forms as described on [this Trello](https://trello.com/c/kmznrFAm). Some new routing has been added.

### How to review 
Check that the new questions and routing match the overflow diagram attached to the trello card.

### CODE REVIEW

**Comment by petechd** [comment]
[[A] NISRA New questions - work (S)](https://trello.com/c/kmznrFAm/3817-a-nisra-new-questions-work-s)

File: source/jsonnet/northern-ireland/individual/blocks/employment/place_of_work.jsonnet [hunk 1]
```diff
@@ -0,0 +1,93 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local ireland = 'No, it is in the Republic of Ireland';
+local pastIreland = 'No, it was is in the Republic of Ireland';
```
Comment (berroar) [inline source/jsonnet/northern-ireland/individual/blocks/employment/place_of_work.jsonnet:5]:
```suggestion
local pastIreland = 'No, it was in the Republic of Ireland';
```

**Comment by MebinAbraham** [comment]
[Quick Launch](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-work-questions/schemas/en/census_individual_gb_nir.json)

File: source/jsonnet/northern-ireland/individual/blocks/employment/place_of_work.jsonnet [hunk 2]
```diff
@@ -0,0 +1,93 @@
+local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local ireland = 'No, it is in the Republic of Ireland';
+local pastIreland = 'No, it was in the Republic of Ireland';
+
+local anotherCountry = 'No, it is in another country';
+local pastAnotherCountry = 'No, it was in another country';
```
Comment (MebinAbraham) [inline source/jsonnet/northern-ireland/individual/blocks/employment/place_of_work.jsonnet:8]: The variable should be named such that it is clear what it is for.

Comment (petechd) [reply source/jsonnet/northern-ireland/individual/blocks/employment/place_of_work.jsonnet]: I named them differently so they are more descriptive now, hopefully.

**Comment by petechd** [comment]
> Last checklist on the card says `Routing from country of work to go to school-travel`. I assume it is meant to say `work-travel`?
> 
> Apart from the minor comment, journey works as expected.

I'm checking this with Donna.

**Pull request merged**